auth/default.go

```diff
@@ -53,7 +53,7 @@ func (n *noop) Generate(id string, opts ...GenerateOption) (*Account, error) {
 		Secret:   options.Secret,
 		Metadata: options.Metadata,
 		Scopes:   options.Scopes,
-		Issuer:   n.Options().Namespace,
+		Issuer:   n.Options().Issuer,
```

Comment:
Why has this changed? It looks perfectly reasonable but just wondering how it's gone from Namespace to Issuer

Reply:
It's to be more inline with the domain based naming, e.g. domain for registry, network for router etc. I feel like issuer is a lot more in-line with what you'd expect for an auth account.